### PR TITLE
Add feature flag for simplify deposits UI.

### LINKED ISSUE #5690: Feature flag for simplify deposits UI
### Description

For simplify deposits UI project: pdjTHR-2i5-p2, implement a feature flag.

### Acceptance criteria

* In PHP, there is a function that will return a boolean whether or not simplify deposits UI feature is enabled.
* There is a way in the frontend / Javascript to determine whether this feature flag is enabled.
* Clear instruction to turn on this feature flag on / off for developer. No need UI to toggle this flag.
* This feature flag has default value off.

### Designs

Not too related to this issue but the here is the project design: paJDYF-7fi-p2

### PR DESCRIPTION
**Pull request by shendy-a8c**
Fixes https://github.com/Automattic/woocommerce-payments/issues/5690.

#### Changes proposed in this Pull Request

Add a feature flag for simplify deposits UI project pdjTHR-2i5-p2.

ps: I intentionally do not add unit test since this feature flag is somewhat temporary. Let me know if I should add a unit test.

#### Testing instructions

* Install [WP Console](https://wordpress.org/plugins/wp-console/).
* To check in PHP, use `WC_Payments_Features::is_simplify_deposits_ui_enabled()`.
* To check in Javascript, open Payments > Overview page and type `window.wcpaySettings.featureFlags.simplifyDepositsUi` in dev console. If feature flag is disabled, it will return non-true value.
* To enable, run `update_option( '_wcpay_feature_simplify_deposits_ui', '1' );` in WP Console. To disable, set option to '0'.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

File: includes/class-wc-payments-features.php [hunk 1]
```diff
@@ -254,6 +264,7 @@ public static function to_array() {
 				'woopayExpressCheckout'   => self::is_woopay_express_checkout_enabled(),
 				'isAuthAndCaptureEnabled' => self::is_auth_and_capture_enabled(),
 				'progressiveOnboarding'   => self::is_progressive_onboarding_enabled(),
+				'simplifyDepoitsUi'       => self::is_simplify_deposits_ui_enabled(),
```
Comment (mattallan) [inline includes/class-wc-payments-features.php]:
```suggestion
				'simplifyDepositsUi'       => self::is_simplify_deposits_ui_enabled(),
```

Small typo :)

**Review by mattallan — APPROVED** [review]
Thanks @shendy-a8c 

I made a note about a small typo but apart from that, these chanegs look good!

> ps: I intentionally do not add unit test since this feature flag is somewhat temporary. Let me know if I should add a unit test.

Sounds fine to me!

**Comment by shendy-a8c** [comment]
Nice catch! Thank you @mattallan and @Jinksi!

**Comment by shendy-a8c** [comment]
Your commits look good, @Jinksi! Thanks.

**Pull request merged**